basis_universal doesn't respect USE_SYSTEM_ZSTD

### Game or games this happens in

None

### What area of the game / PPSSPP

Build time

### What should happen

basis_universal should use the system zstd header

### Logs

```
FAILED: CMakeFiles/basis_universal.dir/ext/basis_universal/basisu_transcoder.cpp.o 
/usr/bin/c++ -DASSETS_DIR=\"/usr/local/share/ppsspp/assets/\" -DGLEW_NO_GLU -DPNG_ARM_NEON_OPT=0 -DSHARED_LIBZIP -DSHARED_ZLIB -DUSE_SYSTEM_MINIUPNPC -DVK_USE_PLATFORM_XLIB_KHR -DWITH_UPNP -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE=1 -D_XOPEN_SOURCE=700 -D_XOPEN_SOURCE_EXTENDED -D__BSD_VISIBLE=1 -D__LIBRETRO__ -D__STDC_CONSTANT_MACROS -I/build/libretro-ppsspp/src/libretro-ppsspp/ext/glslang -I/build/libretro-ppsspp/src/libretro-ppsspp/ext/glew -I/build/libretro-ppsspp/src/libretro-ppsspp -I/build/libretro-ppsspp/src/libretro-ppsspp/ext/openxr -I/build/libretro-ppsspp/src/libretro-ppsspp/Common -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -g -ffile-prefix-map=/build/libretro-ppsspp/src=/usr/src/debug/libretro-ppsspp -flto=auto   -Wformat -Wno-multichar -msse2 -Wno-psabi -fPIC -fno-strict-aliasing -std=gnu++17 -MD -MT CMakeFiles/basis_universal.dir/ext/basis_universal/basisu_transcoder.cpp.o -MF CMakeFiles/basis_universal.dir/ext/basis_universal/basisu_transcoder.cpp.o.d -o CMakeFiles/basis_universal.dir/ext/basis_universal/basisu_transcoder.cpp.o -c /build/libretro-ppsspp/src/libretro-ppsspp/ext/basis_universal/basisu_transcoder.cpp
/build/libretro-ppsspp/src/libretro-ppsspp/ext/basis_universal/basisu_transcoder.cpp:158:26: fatal error: ../zstd/lib/zstd.h: No such file or directory
  158 |                 #include "../zstd/lib/zstd.h"
      |                          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

### Platform

Linux / BSD

### Mobile phone model or graphics card

N/A

### PPSSPP version affected

HEAD

### Last working version

_No response_

### Graphics backend (3D API)

Other

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [x] Try resetting settings or older versions and include if the issue is related.
- [x] Try without any cheats and without loading any save states.
- [X] Include logs or screenshots of issue.

--- a/GPU/Common/ReplacedTexture.cpp
+++ b/GPU/Common/ReplacedTexture.cpp
@@ -259,7 +259,6 @@ void ReplacedTexture::Prepare(VFSBackend *vfs) {
 		return;
 	}
 
-	fmt = fmt;
 	SetState(ReplacementState::ACTIVE);
 
 	if (threadWaitable_)

--- a/ext/basis_universal/basisu_transcoder.cpp
+++ b/ext/basis_universal/basisu_transcoder.cpp
@@ -155,7 +155,7 @@
    // If BASISD_SUPPORT_KTX2_ZSTD is 0, UASTC files compressed with Zstd cannot be loaded.
 	#if BASISD_SUPPORT_KTX2_ZSTD
 		// We only use two Zstd API's: ZSTD_decompress() and ZSTD_isError()
-		#include "../zstd/lib/zstd.h"
+		#include <zstd.h>
 	#endif
 #endif
 

